coordinator/events/eventlog.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Edgeless Systems GmbH.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package events
+
+import (
+	"time"
+	"net/http"
+	"encoding/json"
+)
+
+type ActivationEvent struct {
+	MarbleType	string	`json:"marbleType"`
+	UUID		string	`json:"uuid"`
+	Quote		[]byte	`json:"quote"`
+}
+
+type Event struct {
+	Timestamp	time.Time	`json:"time"`
+	Activation *ActivationEvent `json:"activation"`
+}
+
+type Log struct {
+	events []Event
+}
+
+func NewLog() *Log {
+	return &Log{}
+}
+
+func (ev *Log) Activation(marbleType string, uuid string, quote []byte) {
+	ev.events = append(ev.events, Event{
+		Timestamp:	time.Now(),
+		Activation: &ActivationEvent{MarbleType: marbleType, UUID: uuid, Quote: quote},
+	})
+}
+
+func (ev *Log) Handler() http.HandlerFunc {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+		json.NewEncoder(w).Encode(ev.events)
+	})
+}
```

Comment:
```suggestion
func (l *Log) Activation(marbleType string, uuid string, quote []byte) {
	l.events = append(l.events, Event{
		Timestamp:	time.Now(),
		Activation: &ActivationEvent{MarbleType: marbleType, UUID: uuid, Quote: quote},
	})
}

func (l *Log) Handler() http.HandlerFunc {
	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
		w.Header().Set("Content-Type", "application/json")
		w.WriteHeader(http.StatusOK)
		json.NewEncoder(w).Encode(l.events)
	})
}
```
Just a small style nitpick